e2e/util/namespace.go

```diff
@@ -63,10 +63,16 @@ func DeleteNamespace(ctx types.Context, cluster types.Cluster, namespace string)
 		return nil
 	}
 
-	log.Debugf("Waiting until namespace %q is deleted in cluster %q", namespace, cluster.Name)
+	return nil
+}
 
+func WaitForNamespaceDelete(ctx types.Context, cluster types.Cluster, namespace string) error {
```

Comment:
This is fine - but we should use the new generic wait function add in the next commit.